heft-plugins/heft-sass-plugin/src/SassPlugin.ts

```diff
@@ -26,131 +36,148 @@ const SASS_CONFIGURATION_FILE_SPECIFICATION: ConfigurationFile.IProjectConfigura
     jsonSchemaObject: sassConfigSchema
   };
 
+/**
+ * @public
+ */
+export interface ISassPluginAccessor {
+  /**
+   * Hook that will be invoked after the CSS is generated but before it is written to a file.
+   */
+  readonly postProcessCssAsyncHook: AsyncSeriesWaterfallHook<string>;
```

Comment:
We've followed this pattern in the past: 

```suggestion
  readonly hooks: {
    /**
     * Hook that will be invoked after the CSS is generated but before it is written to a file.
     */
    readonly postProcessCss: AsyncSeriesWaterfallHook<string>;
  }
```